Tests/test_Nexus.py

```diff
@@ -315,11 +323,11 @@ def test_NexusTest1(self):
         """\
 usertype matrix_test stepmatrix=5
         A        C        G        T        -
-[A]     .       2.40     2.57     2.43     2.43     
-[C]    2.40      .       2.28     2.12     2.14     
-[G]    2.57     2.28      .       2.31     2.31     
-[T]    2.43     2.12     2.31      .       2.14     
-[-]    2.43     2.14     2.31     2.14      .       
+[A]     .       2.40     2.57     2.43     2.43
```

Comment:
I don't think you should have changed the whitespace here - this is where TravisCI reports the output does not match this given text:

```
======================================================================
FAIL: test_NexusTest1 (test_Nexus.NexusTest1)
Test Nexus module
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/biopython/biopython/Tests/test_Nexus.py", line 332, in test_NexusTest1
    """)  # noqa for pep8 W291 trailing whitespace
AssertionError: 'usertype matrix_test stepmatrix=5\n        A        C        G        T        -\n[A]     .       2.40     2.57     2.43     2.43     \n[C]    2.40      .       2.28     2.12     2.14     \n[G]    2.57     2.28      .       2.31     2.31     \n[T]    2.43     2.12     2.31      .       2.14     \n[-]    2.43     2.14     2.31     2.14      .       \n;\n' != 'usertype matrix_test stepmatrix=5\n        A        C        G        T        -\n[A]     .       2.40     2.57     2.43     2.43\n[C]    2.40      .       2.28     2.12     2.14\n[G]    2.57     2.28      .       2.31     2.31\n[T]    2.43     2.12     2.31      .       2.14\n[-]    2.43     2.14     2.31     2.14      .\n;\n'
======================================================================
FAIL: test_TreeTest1 (test_Nexus.NexusTest1)
Test Tree module.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/biopython/biopython/Tests/test_Nexus.py", line 375, in test_TreeTest1
    self.assertEqual(len(output.split("\n")), len(expected.split("\n")))
AssertionError: 22 != 21
----------------------------------------------------------------------
```